docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Snapshot
+val sourceETLDB = "ow_bronze_snapshot_etl"
+val targetPrefix = "/mnt/overwatch_playground/721-snap/incremental"
+val pipeline = "Bronze,Silver,Gold"
+val snapshotType = "Incremental"
+val tablesToExclude = "notebook_silver,notebook_gold"
+Snapshot.process(sourceETLDB,targetPrefix,snapshotType)
+```
+
+We need below parameters to run the snapshot process through Databricks Notebook:
+
+| Param           | Type   | Optional | Default Value        | Description                                                                                                                                        |
+|-----------------|--------|----------|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| sourceETLDB     | String | No       | NA                   | Source Database Name for which Snapshot need to be done.                                                                                           |
+| targetPrefix    | String | No       | NA                   | Target path where Snapshot need to be done.                                                                                                        |
+| snapshotType    | String | No       | NA                   | Type of Snapshot to be performed. "Full" for Full Snapshot, "Incremental" for Incremental Snapshot                                                 |
+| pipeline        | String | Yes      | "Bronze,Silver,Gold" | Define the Medallion Layers. Argument should be in form of "Bronze, Silver, Gold"(All 3 or any combination of them)                                |
+| cloneLevel      | String | Yes      | "DEEP"               | Clone Level for Snapshot. By Default it is "DEEP". You can also specify "SHALLOW" Clone                                                            |
+| tablesToExclude | String | Yes      | ""                   | Array of table names to exclude from the snapshot. Tables should be separated with ",". This is the table name only - without the database prefix. |
+
+After snapshot process is done the tables from Source ETL Database will be cloned in the target prefix location. The structure for target prefix location would be like below:
+
+![Snapshot_Target](/images/DataEngineer/snapshotTarget.png)
+
+All the tables data would be there in "targetPrefix/data" folder as shown below:
+
+![Snapshot_Target_Data](/images/DataEngineer/snapshotTarget_Data.png)
+
+If you look into the targetPrefix folder there would one another subdirectory called "clone_report". This folder contains the data regarding the report to show the status of your snapshot process for each table:
+
+![clone_report](/images/DataEngineer/clone_report.png)
+
+##### Snapshot through Databricks Job
+
+Snapshot process can also be configured using Databricks Job. For this we need to run SnapshotRunner Class.
```

Comment:
Change to: "You can also schedule a job to either run the notebook above, or use the SnapshotRunner class. For example: "